### DISCUSSION TITLE
Introducing: GitBook Library 📚

### DISCUSSION THREAD

**Discussion by addisonschultz** [GitbookIO/community#187] [Announcements]
Hello everyone! I'm excited to share a new feature we shipped on our website - [**GitBook Library**](https://www.gitbook.com/gitbook-library)! 

![Twitter-LinkedIn 1 91_1 - 3](https://user-images.githubusercontent.com/42930383/234513363-ababd2f7-6fba-4a31-8d82-f8f4e77c301f.png)

GitBook Library is a place where you can discover, browse, and submit content to be featured on our site—And we'd love to include yours! Submissions are now open, and you can add yours using [this form](https://forms.gle/cmkuQqrHNNjCxyvX8).

We have a ton of improvements and ideas we'd like to add to this, but in the meantime—We hope you can discover something new! 

📚

**Comment by ChoiceBANKsampleritgithub** [comment]
Hello

**Comment by addisonschultz** [comment]
Hello GitBookers! As an update from our side, we've decided to sunset the GitBook Library for the time being, in light of a more updated directory of GitBook sites, automatically generated from the most popular sites across our platform.

You can find the updated page here: https://www.gitbook.com/open/library/ !
